app/models/flood_risk_engine/enrollment_exemption.rb

```diff
@@ -22,6 +22,13 @@ class EnrollmentExemption < ActiveRecord::Base
       operator_failings: 1
     }
 
+    enum assistance_mode: {
+      unassisted: 0,
+      fully: 1,
+      partial: 2,
+      admin_intervention: 3
+    }
+
```

Comment:
yeah been umming and ahhhing whether  to change them, but just went with WEX values n the end for simplicity ..only used in couple of places